.github/actions/setup-dependencies/action.yml

```diff
@@ -15,7 +15,7 @@ runs:
 
     - name: Install sasquatch
       run: |
-        curl -L -o sasquatch_1.0_amd64.deb https://github.com/onekey-sec/sasquatch/releases/download/sasquatch-v4.5.1-1/sasquatch_1.0_amd64.deb
```

Comment:
This line is duplicated in `installation.md` and `Dockerfile` as well

Reply:
fixed